src/client/App.js

```diff
@@ -22,7 +23,7 @@ const App = () => {
         </div>
 
         <div className="four wide column">
-          <MostUsedWords wordInfoList={groupedCountWords(mostFrequentWords(exampleTweets.map((data) => data.tweet)))} search=''/>
+          <MostUsedWords wordInfoList={groupedCountWords(mostFrequentWords(state.feed.map((post) => post.data)))} search=''/>
         </div>
       </div>
```

Comment:
Also can you move the 3 'result's section into their own component and mapStateToProps to that instead of the whole app